march_hardware/src/IMotionCube.cpp

```diff
@@ -0,0 +1,488 @@
+// Copyright 2018 Project March.
```

Comment:
This class is too long, many functions look like helper function and I think they should be moved for readability.
Functions such as: parseStatusWord

for now we can add a todo if you agree